### DISCUSSION TITLE
create an automap model

### DISCUSSION THREAD

**Discussion by Jackovic** [collerek/ormar#145] [Q&A] [answered]
Hello,

I would like to know how to create an automap model ?
The idea is to create an Ormar Model based on an existing table in database. Is it possible, if so how ?
Thanks !

**Comment by collerek** [comment, marked as answer]
Hi,

Right now this is not possible, I plan to write simple translator from `sqlalchemy` to `ormar`, but that will probably be separate small lib/package. With this translator you can use sqlalchemy automap and translate models to ormar from sqlalchemy (to not reinvent the wheel). But this does not exist yet.

**Comment by collerek** [comment]
Ok the beta version of the package to convert sqlalchemy orm models to ormar is up and running: https://github.com/collerek/sqlalchemy-to-ormar
